### ISSUE TITLE
series

### ISSUE THREAD

**Issue by meat-ball** [jamorin/Sonarr.bundle#3]
hey first off, awesome thanks for putting this together. Everything is working well except for Series, where I get "channel not responding". Everything else is on point.
I'll take a look and see if it's something on my end that's preventing the series list from being picked up (I have PMS and Sonarr on two different machines).

**Comment by jamorin** [comment]
What do you see in `PMS Plugin Logs/com.plexapp.plugins.sonarr.log` when you visit the Series page?

**Comment by meat-ball** [comment]
Hey, thanks for following up. When I try series:

 File "/Users/damien/Library/Application Support/Plex Media Server/Plug-ins/Framework.bundle/Contents/Resources/Versions/2/Python/Framework/components/runtime.py", line 843, in handle_request
    result = f(**d)
  File "/Users/damien/Library/Application Support/Plex Media Server/Plug-ins/Sonarr.bundle/Contents/Code/**init**.py", line 61, in Series
    network = series['network']
  File "/Users/damien/Library/Application Support/Plex Media Server/Plug-ins/Framework.bundle/Contents/Resources/Versions/2/Python/Framework/code/sandbox.py", line 108, in <lambda>
    _getitem_           = lambda x, y: x.**getitem**(y),
KeyError: 'network'

2015-03-20 14:50:18,315 (1085ca000) :  DEBUG (runtime:106) - Sending packed state data (107 bytes)
2015-03-20 14:50:18,315 (1085ca000) :  DEBUG (runtime:918) - Response: [500] 809 bytes

**Comment by jamorin** [comment]
Looks like 'network' is missing from one of your shows, I added a check for it in master.

**Comment by meat-ball** [comment]
awesome- works beautifully. Thanks a lot, very appreciated.

**Comment by meat-ball** [comment]
nothing out of the ordiary on the logs. I tried messing around with the
settings as far as URL base (should I list something there? I never have
previously) or using my DNS address insead of my local ip. .

Im getting the same exact scenario with the couchpotato plugin. everyting
works, except adding something new (unless it’s on one of the suggested
lists). It happened as soon as the ios app was updated (least that’s when I
noticed).

nevermind- I just saw your note on github. I assume this is due to the new
apps- PMP, iOS etc?
It works in plex home theater, albeit not really where you would need it.
Im happy to send you my logs if you think it would help but it sounds like
you know where the problem exists.
